### ISSUE TITLE
Easy way to close the server, without restarting sbt?

### ISSUE THREAD

**Issue by danyaljj** [quantifind/wisp#47]
Suppose I plot sth; eg. the following: 
```scala 
Highcharts.areaspline(recall, precision)
Highcharts.xAxis("Recall")
Highcharts.yAxis("Precision")
```
It starts running the server and hangs in there: 
```
[info] INFO  [2016-05-28 16:35:30,037] org.eclipse.jetty.server.Server: jetty-8.1.13.v20130916
[info] INFO  [2016-05-28 16:35:30,066] o.e.jetty.server.AbstractConnector: Started SocketConnector@0.0.0.0:60126
[info] Server started: http://192.168.0.12:60126/index-1464478529881.html
[info] Output written to http://192.168.0.12:60126 (CMD + Click link in Mac OSX).
[info] Output written to http://192.168.0.12:60126 (CMD + Click link in Mac OSX).
[info] Output written to http://192.168.0.12:60126 (CMD + Click link in Mac OSX).
```
How can I stop the plot server without restarting sbt?

**Comment by AustinBGibbons** [comment]
Hi Daniel!

You should be able to run 

```Highcharts.stopServer```

Let me know if it works for you

**Comment by danyaljj** [comment]
hmm what? 😯

**Comment by AustinBGibbons** [comment]
Hi Daniel!

You should be able to run 

```
Highcharts.stopServer
```


Let me know if it works for you

(I think we came across a bug in github's markdown parsing :-)

**Comment by danyaljj** [comment]
Yeah I came cross `Highcharts.stopServer`, but the thing is it closes it right away; and I don't want to close it programmatically. Rather I am looking for an external solution; like a shortcut key (say ctrl+D or whatever). Think about this scenario: As a developer I change sth in the code, run it, and visualize the result; then stop the visualization and repeat the process. For developer who need to go through this cycle, restarting sbt each time is such a big waste of time. If there is no way of closing the visualization manually (like using some sort of shortcuts, rather than closing it programmatically) I would rather switch to another tool.

**Comment by AustinBGibbons** [comment]
Hi Daniel!

Why do you want to stop the visualization process? My _modus operandi_ was always to just leave it idly in the background.

Unfortunately wisp is not under active development, and was more of a one-time push, but it probably wouldn't be too hard to extend it to capture sigterm (which I believe is the signal ctrl+D sends) and have it call stopServer
